kernel/README.md

```diff
@@ -2,7 +2,11 @@
 
 This document explains the steps to build a compatible kernel with Kata
 Containers. To do this use build-kernel.sh, this script automates the
-process to build a kernel for Kata Containers. 
```

Comment:
Nitpick - but I think you've accidentally added an extra space (` `) to the end of this line - I was wondering why it turned up in the patch ;-)